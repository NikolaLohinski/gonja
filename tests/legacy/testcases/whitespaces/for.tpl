##########
{% for x in [1,2,3] %}
{{ x }}
{% endfor %}
##########
{% for x in [1,2,3] -%}
{{ x }}
{% endfor %}
##########
{% for x in [1,2,3] %}
{{ x }}
{%- endfor %}
##########
{% for x in [1,2,3] -%}
{{ x }}
{%- endfor %}
##########
{% for x in [1,2,3] %}
{{- x }}
{% endfor %}
##########
{% for x in [1,2,3] %}
{{- x -}}
{% endfor %}
##########
{% for x in [1,2,3] -%}
{{ x }}
{%- if x is even -%}
(is modulo)
{%- endif %}
{% endfor %}
##########