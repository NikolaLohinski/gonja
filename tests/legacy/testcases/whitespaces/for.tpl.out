##########

1

2

3

##########
1
2
3

##########

1
2
3
##########
123
##########
1
2
3

##########
123
##########
1
2(is modulo)
3

##########